optaplanner-wb-screens/optaplanner-wb-solver-editor/optaplanner-wb-solver-editor-backend/src/main/java/org/optaplanner/workbench/screens/solver/backend/server/ToSolverConfig.java

```diff
@@ -116,4 +122,23 @@ private TerminationConfig create( final TerminationConfigModel termination ) {
             return terminationConfig;
         }
     }
+
+    private List<PhaseConfig> create( final List<PhaseConfigModel> phaseConfigList ) {
+        if ( phaseConfigList == null ) {
+            return null;
```

Comment:
ToSolverConfigModel.create returns an empty list, this returns null. Wouldn't it be better to return an empty list as well?